source/dataman/HDF5/AMHDF5File.cpp

```diff
@@ -199,44 +220,167 @@ bool AMHDF5File::openGroup(const QString &groupName)
 		return true;
 	}
 
-	return false;
+	else {
+		AMErrorMon::alert(this, AMHDF5FILE_GROUP_ALREADY_ADDED, QString("Did not open %1 because %2 has already opened it.").arg(groupName).arg(name_));
+		return false;
+	}
 }
 
-AMHDF5Group* AMHDF5File::findOpenGroup(const QString &groupName) const
+AMHDF5Group* AMHDF5File::groupByName(const QString &groupName) const
```

Comment:
This may not matter but this change may not signify clearly enough that it only queries for open groups.